programs/sll_partition.py

```diff
@@ -0,0 +1,58 @@
+# in a sll make a partition to around a value x. All nodes less than x  come before that all nodes equal or greater to x
+# the x value can appear any where in the right partition. x not needs to apear in the middle, just in right partition.
+
+
+class Node:
```

Comment:
Same as others, use a common `Node` class declared somewhere else. 
In interviews, you are not going to define the node every question